libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -25,29 +25,27 @@ namespace{
 	bool enableDataPath = true;
 
 	//--------------------------------------------------
-//	MARK: - near future
-//	of::filesystem::path defaultDataPath(){
-	std::string defaultDataPath(){
+	of::filesystem::path defaultDataPath(){
 	#if defined TARGET_OSX
 		try{
-			return of::filesystem::canonical(ofFilePath::getCurrentExeDir() / of::filesystem::path("../../../data/")).string();
+			return of::filesystem::canonical(ofFilePath::getCurrentExeDirFS() / of::filesystem::path("../../../data/"));
 		}catch(...){
-			return (ofFilePath::getCurrentExeDir() / of::filesystem::path("../../../data/")).string();
+			return (ofFilePath::getCurrentExeDirFS() / of::filesystem::path("../../../data/"));
 		}
 	#elif defined TARGET_ANDROID
 		return string("sdcard/");
```

Comment:
should this be a fs return not a string?

Reply:
Yes it can be a string. 
my reasoning about using fs: I think this is not part of the public API, 
so the core can use fs::path instead of string, this would avoid windows wstring issues.